set.mm

```diff
@@ -18693,8 +18698,18 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
 
   $( Closed form of Theorem 1977.23 of [Margaris] p. 90.  See ~ 19.23 .
```

Comment:
Theorem 1977.23? I think it must be 19.23...

Reply:
fixed